splunk_eventgen/eventgen_core.py

```diff
@@ -96,15 +97,22 @@ def _load_config(self, configfile, **kwargs):
                 new_args["verbosity"] = args.verbosity
         self.config = Config(configfile, **new_args)
         self.config.parse()
+        if self.config.outputCounter is None and getattr(args, "counter_output"):
```

Comment:
I'm seeing failures here when attempting to launch an eventgen service (args doesn't exist). Maybe use "new_args"?